packages/shared/src/utils/routes/routes.ts

```diff
@@ -53,6 +53,7 @@ export const routes = {
     // cashier_offramp: '/cashier/off-ramp',
     cashier_onramp: '/cashier/on-ramp',
     cashier_p2p: '/cashier/p2p',
+    cashier_p2p_v2: '/cashier/p2p-v2',
```

Comment:
nit: i know p2p is under cashier but eventually we will remove it from under cashier and into wallets, could we have it as its own separate route? just /p2p

Reply:
we need to plan that, then I can change the implementation. 
not sure what should be the correct implementation for that Idea.

Reply:
alright i think better we keep this for now, and in future we can replace this 🙏